generated/nidcpower/nidcpower/session.py

```diff
@@ -1018,6 +1018,238 @@ class _SessionBase(object):
     If you initialize NI-DCPower with a logical name, this property contains the resource descriptor that corresponds to the entry in the IVI Configuration utility.
     If you initialize NI-DCPower with the resource descriptor, this property contains that value.
     '''
+    lcr_actual_load_reactance = _attributes.AttributeViReal64(1150271)
+    '''Type: float
+
+    Specifies the actual reactance, in ohms, of the load used for load LCR compensation.
+    This property applies when lcr_open_short_load_compensation_data_source is set to LCROpenShortLoadCompensationDataSource.AS_DEFINED.
+
+    Note:
+    This property is not supported on all devices. For more information about supported devices, search ni.com for Supported Properties by Device.
+
+    Tip:
+    This property can be set/get on specific channels within your :py:class:`nidcpower.Session` instance.
+    Use Python index notation on the repeated capabilities container channels to specify a subset.
+
+    Example: :py:attr:`my_session.channels[ ... ].lcr_actual_load_reactance`
+
+    To set/get on all channels, you can call the property directly on the :py:class:`nidcpower.Session`.
+
+    Example: :py:attr:`my_session.lcr_actual_load_reactance`
+    '''
+    lcr_actual_load_resistance = _attributes.AttributeViReal64(1150270)
+    '''Type: float
+
+    Specifies the actual resistance, in ohms, of the load used for load LCR compensation.
+    This property applies when lcr_open_short_load_compensation_data_source is set to LCROpenShortLoadCompensationDataSource.AS_DEFINED.
+
+    Note:
+    This property is not supported on all devices. For more information about supported devices, search ni.com for Supported Properties by Device.
+
+    Tip:
+    This property can be set/get on specific channels within your :py:class:`nidcpower.Session` instance.
+    Use Python index notation on the repeated capabilities container channels to specify a subset.
+
+    Example: :py:attr:`my_session.channels[ ... ].lcr_actual_load_resistance`
+
+    To set/get on all channels, you can call the property directly on the :py:class:`nidcpower.Session`.
+
+    Example: :py:attr:`my_session.lcr_actual_load_resistance`
+    '''
+    lcr_load_compensation_enabled = _attributes.AttributeViBoolean(1150222)
+    '''Type: bool
+
+    Specifies whether to apply load LCR compensation data to LCR measurements.
+    Both the lcr_open_compensation_enabled and lcr_short_compensation_enabled properties must be set to True in order to set this property to True.
+
+    Use the lcr_open_short_load_compensation_data_source property to define where the load compensation data that is applied to LCR measurements comes from.
+
+    Load compensation data are applied only for those specific frequencies you define with perform_lcr_load_compensation;
+    load compensation is not interpolated from the specific frequencies you define and applied to other frequencies.
```

Comment:
In the C API reference, this is present as a "Note". Why not do the same here?

Reply:
I've moved this into the "Note" section in my latest commit